src/site/markdown/parameters.md

```diff
@@ -32,6 +32,7 @@ This documents contains various configuration parameters supported by cache engi
 | `-Dmaven.build.cache.lazyRestore=(true/false)`             | Restore artifacts from remote cache lazily                                                                                               | Performance optimization |
 | `-Dmaven.build.cache.restoreGeneratedSources=(true/false)` | Do not restore generated sources and directly attached files                                                                             | Performance optimization |
 | `-Dmaven.build.cache.alwaysRunPlugins=<list of plugins>`   | Comma seprated list of plugins to always run regardless of cache state. An example: `maven-deploy-plugin:*,maven-install-plugin:install` | Remote cache setup/tuning/troubleshooting |
+| `-Dmaven.build.cache.skipLookup=(true/false)`              | Skip looking up artifacts in caches. Does not affect writing artifacts to caches, disables only reading when set to `true`               | May be used to trigger a forced rebuild when maching artifatcs do exist in caches|
```

Comment:
👍

Reply:
@AlexanderAshitkin renamed to -Dorg.maven.build.cache.skipCache